denonavr/volume.py

```diff
@@ -55,6 +55,11 @@ class DenonAVRVolume(DenonAVRFoundation):
         "_volume": "./MasterVolume/value",
         "_muted": "./Mute/value"}
 
+    def __attrs_post_init__(self) -> None:
+        """Initialize the callbacks"""
+        self._device.telnet_api.register_callback("MV", self._volume_callback)
+        self._device.telnet_api.register_callback("MU", self._mute_callback)
```

Comment:
Please register callbacks in `setup` method instead.

Reply:
I originally had it there but that doesn’t seem to work for zones 2 and 3.

Reply:
Did you call `async_setup` for each zone?

Reply:
Your logic (I didn't dig in too deeply) seemed to purposely not do that. Perhaps I'm wrong but it seems to be intentional that this is called for only zone 1